MovieApp/MovieApp/Model/MovieManager.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  MovieManager.swift
+//  MovieApp
+//
+//  Created by Julio Gabriel Tobares on 14/04/2022.
+//
+
+import Foundation
+
+protocol MovieManagerDelegate {
```

Comment:
Move this to Protocols file in Supporting

Reply:
Protocol moved to Supporting